### PR TITLE
Lj reco effi - Corrections

### PR DESCRIPTION
**Pull request by mariajose2398**
- Corrections are made to some of the histograms.
- I renamed some of the references for the selections so that they match the selection name.
- A few new selections and cuts are added.
- More plots for 2D efficiencies  are added

### CODE REVIEW

File: sidm/tools/sidm_processor.py [hunk 1]
```diff
@@ -307,7 +307,7 @@ def build_cuts(self):
     def build_histograms(self):
         """Create dictionary of Histogram objects"""
         hist_menu = utilities.load_yaml(self.histograms_cfg)
-
+
```
Comment (uvabtl) [inline sidm/tools/sidm_processor.py:310]: Hi @mariajose2398, can you fix this extra whitespace real quick before I merge?

Comment (mariajose2398) [reply sidm/tools/sidm_processor.py]: Just did it

**Comment by uvabtl** [comment]
Great, thanks @mariajose2398 ! I'll rebase this now

**Comment by btcardwell** [comment]
Oops, didn't realize I was logged in as uvabtl. Actually rebasing now...

**Pull request merged**